### PR TITLE
perf: Specialized bindings

### LINKED ISSUE #10241: [Perf]Microsoft.UI.Xaml.Controls.BorderLayerRenderer.InnerCreateLayers(...) is slow on Android
### Current behavior

We need to write specialized bindings for Brush.GetBackgroundDrawable(...) / BorderLayerRenderer.GetOverlayDrawable(...) as they consume over 6.65s of CPU time in some apps.

image

image

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

_No response_

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

Android

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### LINKED ISSUE #10253: [Perf]UIElement.AdjustCornerRadius(Canvas, CornerRadius) is slow on Android
### Current behavior

We need to write a specialized binding for UIElement.AdjustCornerRadius(...) as it consumes over 692ms of CPU time in some apps.

image

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

_No response_

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

Android

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by ebariche**
Fixes #10241, Fixes #10253

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]
Very nice!

**Review by jeromelaban — APPROVED** [review]
🚀

File: src/Uno.UI/UI/Xaml/UIElement.Android.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using Uno.UI;
+using Uno.UI;
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/UIElement.Android.cs:1]: Is BOM character removed here?

File: src/Uno.UI/UI/Xaml/Media/Brush.Android.cs [hunk 2]
```diff
@@ -1,9 +1,10 @@
-﻿using System;
+using System;
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Media/Brush.Android.cs:1]: Same question here

File: src/Uno.UI/UI/Xaml/CornerRadius.Android.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/CornerRadius.Android.cs:1]: Same question here

File: src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-﻿#pragma warning disable 0618 // Used for compatibility with SetBackgroundDrawable and previous API Levels
+#pragma warning disable 0618 // Used for compatibility with SetBackgroundDrawable and previous API Levels
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs:1]: Same here

Comment (ebariche) [reply src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs:1]: /azp run

**Pull request merged**